index.html

```diff
@@ -112,140 +137,148 @@ <h3>Multi-Post Stories</h3>
               </div>
             </div>
 
-            <div class="work-2">
-              <div class="work-detail">
-                <div class="main-detail">
-                  <h4>professional art printing data</h4>
-                  <p class="paragraph">
-                    A daily selection of privately personalized reads; no
-                    accounts or sign-ups required. has been the industry's
-                    standard
-                  </p>
-                </div>
-                <div class="work-tools work-stack">
-                  <ul>
-                    <li>html</li>
-                    <li>bootstrap</li>
-                    <li>Ruby</li>
-                  </ul>
-                </div>
-                <div class="see-project">
-                  <button class="large-button">see project</button>
+            <div class="all-works__detail">
```

Comment:
- [ ] As the previous code reviewer mentioned, kindly implement the hover effect on each card of your portfolio. The hover should hide the text and the dark opacity and the 'See project' button should appear so the user can click and see the popup window of the project.

Reply:
@leonardodiasb Since both are background images, I mean the linear gradient and the actual image, could you give me an idea to implement these changes on hover?